server/bin/docker/genCerts.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+WORKING="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+. ${WORKING}/deploy-env.sh
+
+PASSPREFIX=pass:
+KEYPASS=password
+PASSWORD=$PASSPREFIX$KEYPASS
+
+if [ ! -d ${SRC_DIR}/pki ]; then
+	mkdir ${SRC_DIR}/pki
+	pushd ${SRC_DIR}/pki
+
+	#Create a private key
+
+	openssl genrsa -des3 -out CA.key -passout ${PASSWORD} 4096
```

Comment:
password variable should be quoted (suggest passing whole script through shellcheck)

Also, it's generally safer to use the env:  prefix for passwords, and put the password in the environment. That way, it's not visible to other users who can list processes.